crates/sui-storage/src/indexes.rs

```diff
@@ -705,12 +705,22 @@ impl IndexStore {
             TxSequenceNumber::MIN
         });
 
-        let key = (
-            package,
-            module.clone().unwrap_or_default(),
-            function.clone().unwrap_or_default(),
-            cursor_val,
-        );
+        const MAX_STRING: &str =
```

Comment:
Where should we store this? 

Can also change this to load dynamically `max_type_nodes` value.